### ISSUE TITLE
pretrained model prediction result

### ISSUE THREAD

**Issue by BinghaoLu** [openclimatefix/skillful_nowcasting#46]
Hi, I just used the pretrained model you provided to evaluate on the sample dataset, the result doesn't seem to make reasonable prediction, I also tried to train the model by myself, but hasn't good prediction either, has anyone used the pretrained model to predict good result, or have you successfully train the model to make good result?

**Comment by Cirrusfloccus31** [comment]
Hi,

I think I have the same problem. Even if the input is a torch.zeros the model output has no sense !
Did someone has good results with the pretrained model ?

Thank you

**Comment by BinghaoLu** [comment]
I added sigmoid activation to the output during the training and evaluation and it produced reasonable results.
